### ISSUE TITLE
Add support for Windows, mac and Web

### ISSUE THREAD

**Issue by rupamking1** [rikulo/socket.io-dart#28]
Add support for Windows, mac and Web

**Comment by jumperchen** [comment]
This server side library can work on Windows, Mac, and Linux env. which is the same as Dart VM.

**Comment by rupamking1** [comment]
Can I use this library as Client in Windows, mac and Web

**Comment by jumperchen** [comment]
No, for socket.io-client, you should use https://github.com/rikulo/socket.io-client-dart.
This library is used as a server to manage all socket client connections.
